src/miner.cpp

```diff
@@ -473,6 +518,137 @@ void BlockAssembler::addScoreTxs(std::vector<const CTxMemPoolEntry *> *vtxe)
     }
 }
 
+// Skip entries in mapTx that are already in a block.
+bool BlockAssembler::SkipMapTxEntry(CTxMemPool::txiter it)
+{
+    assert(it != mempool.mapTx.end());
+    if (inBlock.count(it))
+        return true;
+    return false;
+}
+
+void BlockAssembler::SortForBlock(const CTxMemPool::setEntries &package, std::vector<CTxMemPool::txiter> &sortedEntries)
```

Comment:
Same comment as below:  "Good point but we can't because we still have both in our regression tests such as ctor.py and also in the miner we later either do the sort or not based on the "canonical" flag. I think if we wan't to remove support for TTOR then it should be done in a different PR."

Reply:
I agree, let's leave TTOR around for a while longer.